src/commonmark/en/content/implementation/support.md

```diff
@@ -103,3 +106,23 @@ information as possible:
   - Problem description: Describe the problem clearly, why you think it
     is a problem and which behavior you would expect from the system.
 
+Your bug report will be investigated by the developer team and be given a status. 
+If valid it will also be assigned to a developer and approver and eventually be
+fixed. Note that bugfixes are incorporated into the trunk and latest
+release branch - so more testing and feedback to the developer teams
+leads to higher quality of your software.
+
+If you want to suggest new functionality to be implemented in DHIS2 you
+should first start a discussion on the Community of Practice to get feedback on your 
+idea and confirm that the functionality you are suggesting does not already exist. 
+Once you have completed these steps, you can submit a feature request on DHIS2 Jira 
+by clicking "Create" in the top menu and selecting "Feature" as the issue type. 
```

Comment:
I wonder if this is a good place (and time) to try to introduce the "User Story" tickets instead of feature. I think Markus and Anne were developing a template. Maybe we put that here and try to encourage that way of requesting new features...?